docs/docusaurus.config.ts

```diff
@@ -61,7 +61,7 @@ const config: Config = {
       {
         versions: [
           {
-            guidesPath: "./current_docs/guides",
```

Comment:
Note: if this PR is merged before #6869, some tutorials will be hidden as they will be in a different location.